evcxr_jupyter/samples/evcxr_jupyter_tour.ipynb

```diff
@@ -1,5 +1,31 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Google Colab Rust Setup\n",
+    "\n",
+    "The following cell is used to set up and spin up a Jupyter Notebook environment with a Rust kernel using Nix and IPC Proxy. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!wget -qO- https://gist.github.com/wiseaidev/2af6bef753d48565d11bcd478728c979/archive/3f6df40db09f3517ade41997b541b81f0976c12e.tar.gz | tar xvz --strip-components=1\n",
```

Comment:
Would it be possible to put the setup script in the evcxr repository? That would mean that if the file needs editing, we can just edit it via a regular github PR. Ideally the script could just live in the same directory as this file, although that would presumably only work if colab checks out the whole repository when it loads an ipynb file from github, but hopefully that's what it does.

Reply:
Got it. But, it turns out it only copies the ipynb file and not the whole directory:
![Screenshot from 2023-12-20 23-50-50](https://github.com/evcxr/evcxr/assets/62179149/81b1d2ed-ac1e-490d-8ef7-bf67de1ae873)

The script lives in [this gist](https://gist.github.com/wiseaidev/2af6bef753d48565d11bcd478728c979) that I currently maintain. I will try my best to keep it up to date!